lib/tasks/vmodl_helper.rb

```diff
@@ -159,6 +148,48 @@ def dump_vmodl!
 
   private
 
+  def build_wsdl_obj!(type)
+    puts "Adding #{type_name} to vmodl"
```

Comment:
Is this debug, or do you want this merged?

Reply:
Oh nvm - I didn't see that it was in the original code.

Reply:
Yeah it is nice to see what is being added when running the rake task